packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -150,6 +150,28 @@ export class W3mAccountDefaultWidget extends LitElement {
     `
   }
 
+  private orderedFeaturesTemplate() {
+    const featuresOrder = this.features?.experimental_walletFeaturesOrder || [
+      'onramp',
+      'swaps',
+      'receive',
+      'send'
+    ]
```

Comment:
Can we store these values in a util file ?

Reply:
Yeah they are in `core` constants, will update